components/collector/tests/source_collectors/sonarqube/test_security_warnings.py

```diff
@@ -151,3 +155,15 @@ async def test_filter_security_warnings_hotspots_by_status(self):
             entities=self.hotspot_entities,
             landing_url="https://sonarqube/project/security_hotspots?id=id&branch=master",
         )
+
+    async def test_filter_security_warning_vulnerabilities_by_tag(self):
+        """Test that the security warning vulnerabilities can be filtered by tag."""
+        self.set_source_parameter("tags", ["bug"])
```

Comment:
Would it not make more sense to include by "other tag", to verify it only includes one?

Reply:
Yes, if the collector did the filtering, but in this case the filtering is done by SonarQube. We just pass the query parameter. I'll add an assert that the GET method is called with the correct query.

Reply:
And extend the other tests as well.